src/test/java/uk/gov/pay/adminusers/resources/ServiceUpdateOperationValidatorTest.java

```diff
@@ -15,23 +15,11 @@
 
 public class ServiceUpdateOperationValidatorTest {
 
+    private static final String GO_LIVE_STAGE_INVALID_ERROR_MESSAGE = "Field [value] must be one of [NOT_STARTED, ENTERED_ORGANISATION_NAME, CHOSEN_PSP_STRIPE, CHOSEN_PSP_WORLDPAY, CHOSEN_PSP_SMARTPAY, CHOSEN_PSP_EPDQ, TERMS_AGREED_STRIPE, TERMS_AGREED_WORLDPAY, TERMS_AGREED_SMARTPAY, TERMS_AGREED_EPDQ, DENIED, LIVE]";
+    
     private final ObjectMapper mapper = new ObjectMapper();
     private final ServiceUpdateOperationValidator serviceUpdateOperationValidator = new ServiceUpdateOperationValidator(new RequestValidations());
 
-    @Test
```

Comment:
These 2 tests I've just moved down the class a bit so that the common tests are grouped together at the top followed by the service name tests. They're now at line 53